build-check.txt

```diff
@@ -0,0 +1,2 @@
+Integrity checks failed:
+#<struct Commons::Integrity::Check::Base::Error category=:position_id_message, message="Q48730651 was found in boundaries/build/index.json but not in boundaries/build/position-data.json", filename=#<Pathname:boundaries/build/position-data.json>>
```

Comment:
Is this something we should do something about?

Reply:
Yes. Probably a good idea, just for consistency's sake. It's probably an incomplete-data-in-WD issue, which I'll look into.

Reply:
Fixed :-)

Reply:
:shipit: